hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1016,6 +1016,7 @@ public OzoneFileStatus getOzoneFileStatus(String volumeName,
         .setBucketName(bucketName)
         .setKeyName(keyName)
         .setRefreshPipeline(true)
+        .setSortDatanodesInPipeline(topologyAwareReadEnabled)
```

Comment:
@rakeshadr , can you plz add some details here why sorting of dns is required here?

Reply:
Thanks @bshashikant for the reviews. This change is basically to make "getFileStatus" and "listStatus" apis similar to "lookupFile()" API, where it does sorting.

Reference:
https://github.com/apache/hadoop-ozone/pull/1164#discussion_r451122204